package.json

```diff
@@ -80,12 +80,19 @@
     "production": [
       ">0.2%",
       "not dead",
-      "not op_mini all"
+      "not ie 11",
```

Comment:
TIL this is not included in `not dead` yet. Excluding it anyway because the CSS will be so broken that the site is barely usable in IE 11, so we might as well break even JS on it and show the unsupported browser message